main.go

```diff
@@ -72,37 +72,40 @@ func createInstallNpmCommand() (*command.Model, error) {
 		return nil, fmt.Errorf("unsupported operating system: %s", runtime.GOOS)
 	}
 
-	return command.New(args[0], args[1:]...), nil
+	return command.NewWithStandardOuts(args[0], args[1:]...), nil
 }
 
-func runAndLog(cmd *command.Model) (string, error) {
-	out, err := cmd.RunAndReturnTrimmedCombinedOutput()
+func runAndLog(cmd *command.Model) error {
 	log.Donef(fmt.Sprintf("$ %s", cmd.PrintableCommandArgs()))
-	if err != nil {
-		return out, fmt.Errorf("error running npm command: %s", err)
+	if err := cmd.Run(); err != nil {
+		return fmt.Errorf("error running npm command: %s", err)
 	}
 
-	log.Printf(out)
-
-	return out, nil
+	return nil
 }
 
-func setNpmVersion(ver string) (string, error) {
-	cmd := command.New("npm", "install", "-g", fmt.Sprintf("npm@%s", ver))
-	out, err := runAndLog(cmd)
+func setNpmVersion(ver string) error {
+	cmd := command.NewWithStandardOuts("npm", "install", "-g", fmt.Sprintf("npm@%s", ver))
+	err := runAndLog(cmd)
 	if err != nil {
-		return out, fmt.Errorf("error running npm install: %s", err)
+		return fmt.Errorf("error running npm install: %s", err)
 	}
 
-	return out, nil
+	return nil
 }
 
 func systemDefined() (string, error) {
 	if path, err := exec.LookPath("npm"); err == nil {
 		log.Printf("npm found at %s", path)
 
 		cmd := command.New("npm", "--version")
-		return runAndLog(cmd)
+		log.Donef(fmt.Sprintf("$ %s", cmd.PrintableCommandArgs()))
```

Comment:
I think it would be better to not to mix runAndLog and running and logging in a different way on each location.

Reply:
I personally wouldn't create a new function to only include one line log in it as an extra.